draft-ietf-webtrans-http3.md

```diff
@@ -598,6 +602,33 @@ streams.  A QUIC stream's data limit controls the amount of data that can be
 sent on that stream, WebTransport or otherwise (see {{Section 4
 of !RFC9000}}).
 
+## Negotiating the Use of Flow Control {#flow-control-negotiate}
+
+The use of flow control is optional and can be negotiated.  A WebTransport
+endpoint that allows a WebTransport session to share an underlying transport
+connection with other WebTransport sessions MUST enable flow control.  This
+prevents an application from consuming excessive resources on a
+single session and starving traffic for other sessions (see
+{{security-considerations}}).
+
+An endpoint indicates that it is willing to support more than one WebTransport
+session, and thus flow control, by sending the SETTINGS_WT_MAX_SESSIONS with a
+value greater than "1".  If either endpoint sends SETTINGS_WT_MAX_SESSIONS with
+a value of "1", flow control is not enabled, and clients MUST NOT attempt to
+establish more than one simultaneous WebTransport session.  A server that
+receives more than one session on an underlying transport connection when flow
+control is not enabled MUST reset the excessive CONNECT streams with a
+`H3_REQUEST_REJECTED` status (see{{flow-control-limit-sessions}}).
```

Comment:
Some of this feels redundant with the meaning of SETTINGS_WT_MAX_SESSIONS?  Eg: if you said max sessions 1 and you get two simultaneous sessions, that's an error which is orthogonal to the fact that flow control was off.